hydra-explorer/hydra-explorer.cabal

```
@@ -0,0 +1,97 @@
+cabal-version: 3.0
+name:          hydra-explorer
+version:       0.15.0
```

Could: use latest hydra version (as long as the explorer is in the same repo / tagged together with `hydra-node`)

```suggestion
version:       0.16.0
```